packages/protocol-kit/src/utils/eip-3770/config.ts

```diff
@@ -9,11 +9,13 @@ export const networks: NetworkShortName[] = [
   { chainId: 3, shortName: 'rop' },
   { chainId: 4, shortName: 'rin' },
   { chainId: 5, shortName: 'gor' },
+  { chainId: 8, shortName: 'ubq' },
   { chainId: 10, shortName: 'oeth' },
   { chainId: 11, shortName: 'meta' },
   { chainId: 12, shortName: 'kal' },
```

Comment:
There was an interesting use-case that was not checked in the script.
I added chainId=8 instead of chainId=18. The length of both local and safe-deployments configs were the same so the error was not triggered.
I fixed the issue and improved the script so not it is in 100% sync